static/src/javascripts/bootstraps/commercial.js

```diff
@@ -75,6 +77,9 @@ define([
         ['cm-ready', function () {
             mediator.emit('page:commercial:ready');
             userTiming.mark('commercial end');
+            robust.catchErrorsAndLog('ga-user-timing-commercial-end', function () {
```

Comment:
we've got per-module commercial timings, and timings over the commercial bootstrap too. I can see the value in having one place to see all timings, and I can also see a concern about sending out a lot of monitoring calls from the client. I don't mind which you choose, just highlighting it in case you decided to log commercial for our team's benefit.